examples/simpack_xrd_dyn_AlGaAs.py

```diff
@@ -0,0 +1,178 @@
+# This file is part of xrayutilities.
+#
+# xrayutilities is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Copyright (c) 2016-2023 Dominik Kriegner <dominik.kriegner@gmail.com>
+
+
+
+# This is an example of the modelling and fitting of the diffraction data obtained on epitaxial heterostructure
```

Comment:
If possible try to keep a 80 character line length. I think the linter will complain otherwise anyways.

Reply:
Done !